addons/survey/wizard/survey_invite.py

```diff
@@ -165,6 +165,14 @@ def _compute_body(self):
                 invite = invite.with_context(lang=langs.pop())
             super(SurveyInvite, invite)._compute_body()
 
+    @api.depends('template_id')
+    def _compute_attachment_ids(self):
```

Comment:
Could you add a small docstring like 
> Used as "onchange" when selecting a template, not meant to update records when new attachments are added to templates on existing wizards.

So we understand why the single dependency.